devtools/platform-descriptor-json/src/main/resources/bundled-codestarts/buildtool/gradle/base/gradle.tpl.qute.properties

```diff
@@ -1,6 +1,6 @@
 #Gradle properties
-#Tue Jun 16 08:41:21 UTC 2020
-quarkusPluginVersion={quarkus.plugin.version}
+quarkusPluginId={quarkus.gradle-plugin.id}
+quarkusPluginVersion={quarkus.gradle-plugin.version}
```

Comment:
is this just to handle we can point to community version when productized versoin not available ?

Reply:
yes! but also because it's makes sense to have it independent from the maven one..

Reply:
@maxandersen test ping comment